### ISSUE TITLE
Missing hyperlink on documentation images

### ISSUE THREAD

**Issue by parulin** [QubesOS/qubes-issues#10223]
### Brief summary

The images of the old documentation were previously also a link, allowing the user to zoom on small devices. It is not the case anymore.

### Steps to reproduce

1. Open https://doc.qubes-os.org/en/latest/introduction/screenshots.html
2. Hover or click on the first image

### Expected behavior

A link is available, opening https://doc.qubes-os.org/en/latest/_images/r4.0-xfce-desktop.png

### Actual behavior

Nothing happens.

### Additional information

This was discussed in #8180: https://github.com/QubesOS/qubes-issues/issues/8180#issuecomment-3188568069

The current options are:

* use `:width:` attribute on every images
* use a custom builder like this one: https://codeberg.org/parulin/doqulint-builder

**Comment by unman** [comment]
My preference would be to use a standard attribute, width, rather than a custom builder.
This will keep the source standards compliant and will be easier for users who wish to build the site locally. (We should imo keep custom requirements to a minimum.)

**Comment by parulin** [comment]
I'm de facto using `:width: 100%`

**Comment by parulin** [comment]
I think that in order to resolve the issue we also need to update the how-to/style guide sections about images to tell the contributors to use `:width:` to create a link when the image might be too large.

**Comment by unman** [comment]
I prefer to explicitly render a link where needed.
